x-pack/plugin/src/test/resources/rest-api-spec/test/watcher/put_watch/90_put_watch_content_validation.yml

```diff
@@ -0,0 +1,12 @@
+---
+"Test empty body is rejected by put watch":
```

Comment:
Maybe it makes sense to just add this test case to 10_basic, since it doesn't require any special setup and just tests a general error edge case. But adding another test file like this is also fine I guess, mostly a matter of taste.

Reply:
Ah ok. I'm still learning the layout of the code directories. I'll make the change you suggested.